### ISSUE TITLE
If I want to redesign a protein...

### ISSUE THREAD

**Issue by haonanzc** [RomeroLab/RLXF#1]
If I have hundreds of one-mutation activity data for a specific protein, would it be feasible and effective to use RLXF to redesign the protein? Or is this dataset size too small for such an approach?

**Comment by nblalock** [comment]
Does your dataset contain 100 sequence variants with single-point mutations for a given parent sequence?

If so, RLXF would still work well but I would recommend training the reward model differently. You can finetune a pLM like ESM2 to predict function (https://www.nature.com/articles/s41467-024-51844-2, we do have a public Github for a way to do this: https://github.com/RomeroLab/finetuning_esm2/tree/main) or finetune a VAE to predict function (if you can obtain an MSA for your protein using something like jackhmmer) instead of the scripts I used to train an ensemble of fully-connected neural networks. There are also more simple methods like ridge regression. I believe this would work well for designing sequence variants with 1-2, maybe 3 mutations. I would be hesitant to add too many mutations. You could also use SolubleMPNN or something to filter sequences less likely to express if you tried to add more mutations than that
